packages/authorization-agent/test/authorization-agent-test.ts

```diff
@@ -8,10 +8,15 @@ import {
   ReadableAccessAuthorization,
   CRUDApplicationRegistration,
   CRUDSocialAgentRegistration,
-  ReadableWebIdProfile
+  ReadableWebIdProfile,
+  ReadableDataAuthorization,
+  ReadableDataInstance,
+  ReadableDataRegistration,
+  CRUDDataRegistry
 } from '@janeirodigital/interop-data-model';
+import { iterable2array } from '@janeirodigital/interop-utils';
```

Comment:
Rename `iterable2array` to `asyncIterableToArray`, otherwise we could just use `Array.from(iter)`